NodeJS/sample/SampleAgentClient.ts

```diff
@@ -10,34 +9,22 @@ let endpoint: string
 let apiKey: string
 let agentUsername: string
 let agentPassword: string
-let skillWithPriorities: SkillPriority[]
 try {
     endpoint = getValue(Constants.ENDPOINT_KEY, args)
     apiKey = getValue(Constants.API_KEY, args)
 
     agentUsername = getValue(Constants.AGENT_USERNAME_KEY, args)
     agentPassword = getValue(Constants.AGENT_PASSWORD_KEY, args)
-    skillWithPriorities = getAgentSkill()
-    console.log(skillWithPriorities)
+
     main();
 } catch (error) {
     console.log(error)
     process.exit(-1)
 }
 
-function getAgentSkill() {
-    let value = getValue(Constants.AGENT_SKILL_KEY, args)
-    if (isValidSkillsWithPriorities(Constants.AGENT_SKILL_KEY, value)) {
-        return JSON.parse(value)
-    } else {
-        throw new Error(`skill priorities have to be specified in json format`)
-    }
-}
-
 async function createAgent(agentClient: AgentClient) {
     try {
-        let response = await agentClient.createAgentAndStation(agentUsername, agentPassword, skillWithPriorities);
-        console.log('agentObject from createAgent');
+        let response = await agentClient.createAgentAndStation(agentUsername, agentPassword);
```

Comment:
you don't do anything with the response?

Reply:
It is a sample. There is not much to do with the result.